index.js

```diff
@@ -264,8 +264,7 @@ module.exports = class Hyperdrive extends EventEmitter {
     return this.files.createReadStream(opts)
   }
 
-  async download (folder, opts) {
-    if (!folder) folder = '/'
+  async download (folder = '/', opts) {
```

Comment:
if (typeof folder === object ... here also

Reply:
Right, you want i.e. `download({ recursive: false })` to just work, nice!